winch/codegen/src/masm.rs

```diff
@@ -165,6 +192,16 @@ pub(crate) trait MacroAssembler {
     /// Calculate remainder.
     fn rem(&mut self, context: &mut CodeGenContext, kind: RemKind, size: OperandSize);
 
+    /// Compare two numbers and put result in dst.
```

Comment:
```suggestion
    /// Compare two operands and put the result in dst.
```

Reply:
Perhaps could we also add a comment stating that in some architectures(x64), this function will emit a series of instructions and update the destination with the result of the comparison?